2-archetect/bullseye/Dockerfile

```
@@ -0,0 +1,21 @@
+ARG REGISTRY_PREFIX=ybor
+FROM ${REGISTRY_PREFIX}/rust:bullseye AS base
+
+RUN cargo install --force --git https://github.com/archetect/archetect.git archetect --root /usr/local
+
+FROM ${REGISTRY_PREFIX}/rust:bullseye
```

```suggestion
FROM ${REGISTRY_PREFIX}/debian:bullseye
```

you can use the debian container, since all you need is the archetect bin and not the full rust tool chain